shared/middleware/properties_api.rb

```diff
@@ -76,6 +76,8 @@ class PropertiesApi < Sinatra::Base
     parsed_value = PropertyBag.parse_value(request.body.read)
     value = PropertyType.new(decrypted_channel_id, storage_id(endpoint)).set(name, parsed_value, request.ip)
 
+    too_large if value[:status] == 'TOO_LARGE'
+
```

Comment:
I think this should be okay, because value[:status] is different than value['status'], no? I'll give it a try in any case